src/clj_docker_client/formatters.clj

```diff
@@ -212,3 +214,13 @@
    :id     (u/format-id (.id network))
    :scope  (.scope network)
    :driver (.driver network)})
+
+(defn format-inspect-network [^NetworkSettings network-settings]
+  {:ip-address (.ipAddress network-settings)
+   :ip-prefix-len (.ipPrefixLen network-settings)})
+
+;; TODO missing bunch of interesting fields here
+;; See https://github.com/spotify/docker-client/blob/6580ae844a3be96421dfce78707163526c8ba82c/src/main/java/com/spotify/docker/client/messages/ContainerInfo.java#L147
+(defn format-inspect
+  [^ContainerInfo container-info]
+  {:network-settings (format-inspect-network (.networkSettings container-info))})
```

Comment:
Try adding the other attributes. Things like formatting the container state is already available in https://github.com/lispyclouds/clj-docker-client/blob/master/src/clj_docker_client/formatters.clj#L65

Reply:
I started solving this but quickly end up tired of just manually rewriting fields from `.hostConfig` to `:host-config` and so on. Has to be a better way out there than manually converting the fields and locking ourselves down to the output.

What about a function that can format the entire output by going through all the public java fields (or getters I guess), making the kebab-case into dash-separated keywords?

Reply:
Thats actually a good idea. I would look into it.